pages/flashbar/content-permutations.page.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import * as React from 'react';
+import ScreenshotArea from '../utils/screenshot-area';
+import createPermutations from '../utils/permutations';
+import PermutationsView from '../utils/permutations-view';
+import Flashbar, { FlashbarProps } from '~components/flashbar';
+import Link from '~components/link';
+import Box from '~components/box';
+
+const noop = () => void 0;
+
+/* eslint-disable react/jsx-key */
+const permutations = createPermutations<FlashbarProps.MessageDefinition>([
```

Comment:
Split these permutations in its own file as the page was becoming too large for the visual regression tests.